sunpy/map/sources/psp.py

```diff
@@ -0,0 +1,43 @@
+"""
+Solar Orbiter Map subclass definitions.
+"""
+
+from sunpy.map import GenericMap
+
+__all__ = ['WISPRMap']
+
+
+class WISPRMap(GenericMap):
+    """
+    WISPR Map
+
+    The The Wide-field Imager for Parker Solar Probe (WISOR) is a white light
```

Comment:
Typo: (WISOR) > WISPR

Reply:
Also, I am thinking if this is better actually:
The **Wide-field Imager for Solar PRobe Plus (WISPR)** is a heliospheric imager ....